tests/cli.rs

```diff
@@ -1191,6 +1195,41 @@ fn cert_without_key() {
         .stderr(predicates::str::is_empty());
 }
 
+#[cfg(feature = "online-tests")]
+#[test]
+fn use_ipv4() {
+    get_command()
+        .args(&["https://v6.ipv6-test.com/api/myip.php", "--body", "--ipv4"])
+        .assert()
+        .failure();
+
+    get_command()
+        .args(&[
+            "https://v4v6.ipv6-test.com/api/myip.php",
+            "--body",
+            "--ipv4",
+        ])
+        .assert()
+        .stdout(function(|output: &str| {
+            IpAddr::from_str(output.trim()).unwrap().is_ipv4()
+        }))
+        .stderr(predicates::str::is_empty());
+}
+
+#[cfg(feature = "online-tests")]
+#[test]
+fn use_ipv6() {
+    get_command()
+        .args(&["https://v4.ipv6-test.com/api/myip.php", "--body", "--ipv4"])
+        .assert()
+        .success();
```

Comment:
Can we have an ipv6 test similar to the earlier ipv4 test?

```rust
get_command()
    .args(&[
        "https://v4v6.ipv6-test.com/api/myip.php",
        "--body",
        "--ipv6",
    ])
    .assert()
    .stdout(function(|output: &str| {
        IpAddr::from_str(output.trim()).unwrap().is_ipv6()
    }))
    .stderr(predicates::str::is_empty());
```

Reply:
not everyone will be able to use ipv6 locally, it will be fail if does not have ipv6 address

maybe we can keep those test case, add a test case with a feature